_docs_integrate/terminate-relationships.md

```diff
@@ -33,14 +33,15 @@ It is then no longer possible to reactivate the Relationship.
 
 ## Terminate an active Relationship
 
-Terminate an active Relationship with the use case [Terminate Relationship]({% link _docs_use-cases/use-case-transport-terminate-relationship.md %}) with the `relationshipId` as input. The `status` of the Relationship is changed to `"Terminated"`. You receive a `transport.relationshipChanged` [event]({% link _docs_integrate/connector-events.md %}), and the peer as well if they use a Connector. Then no Messages can be sent from either side, this includes Requests, Notifications and sharing Attributes. The peer cannot prevent the termination.
+An active Relationship can be terminated by executing the use case [Terminate Relationship]({% link _docs_use-cases/use-case-transport-terminate-relationship.md %}) providing the `relationshipId` as input. The peer cannot prevent the termination of the Relationship. After the Relationship has been terminated, its `status` is changed to `"Terminated"`. You receive a `transport.relationshipChanged` [event]({% link _docs_integrate/connector-events.md %}), and the peer as well if they use a Connector. Then no [Messages]({% link _docs_integrate/data-model-overview.md %}#message) can be sent from either side, this includes [sending or responding to Requests]({% link _docs_integrate/requests-via-messages.md %}) and [exchanging Attributes]({% link _docs_integrate/attribute-introduction.md %}#attribute-management-options). However, please note that Messages whose `content` is a [Notification]({% link _docs_integrate/data-model-overview.md %}#notification) can still be sent on terminated Relationships. Such Notifications cannot be received directly, but they are queued in case the Relationship is [reactivated](#reactivate-a-terminated-relationship). For example, if certain [Attributes were deleted]({% link _docs_integrate/attribute-introduction.md %}#delete-attributes) or [Attributes were updated by succession]({% link _docs_integrate/attribute-introduction.md %}#update-attributes-by-succession) while the Relationship was terminated, the associated Notifications are transmitted after the reactivation. Therefore, the loss of relevant information is prevented.
```

Comment:
> The peer cannot prevent the termination of the Relationship.

I know you copied it, but it sounds kind of negative. :D How about something like "this can be done without the approval/consent of the peer" or "independent of the peer".

>  After the Relationship has been terminated, its `status` is changed to `"Terminated"`.

After the Relationship has been terminated, its `status` changes to `"Terminated"`.

> You receive a `transport.relationshipChanged` [event]({% link _docs_integrate/connector-events.md %}), and the peer as well if they use a Connector.

Why doesn't the peer receive the event if they are an App user. Also, in most cases we try to avoid using "you".

> Then no [Messages]({% link _docs_integrate/data-model-overview.md %}#message) can be sent from either side, this includes [sending or responding to Requests]({% link _docs_integrate/requests-via-messages.md %}) and [exchanging Attributes]({% link _docs_integrate/attribute-introduction.md %}#attribute-management-options).

Then no [Messages]({% link _docs_integrate/data-model-overview.md %}#message) can be sent from either side. This includes [sending or responding to Requests]({% link _docs_integrate/requests-via-messages.md %}) and [exchanging Attributes]({% link _docs_integrate/attribute-introduction.md %}#attribute-management-options).

> However, please note that Messages whose `content` is a [Notification]({% link _docs_integrate/data-model-overview.md %}#notification) can still be sent on terminated Relationships.

I'd rather formulate it as "are sent" instead of "can be sent", since these Messages are generated automatically and the Identity can't/shouldn't send Messages containing Notifications manually.

Note: I'd love this section to be divided into separate lines, as it makes it easier to read and comment on the markdown file. :)